### DISCUSSION TITLE
无法配置configini，求教

### DISCUSSION THREAD

**Discussion by boseliy** [SeaHOH/GotoX#78] [问答/咨询] [answered]
软件下载后，直接用会显示图一，图二是下载后ini里的样子。然后把自己的appid填进去，ipv4改成ipv6就会出现图三四的类似的错误。请教到底怎么修改。
![1](https://user-images.githubusercontent.com/39125354/39815315-23bff7ec-5387-11e8-8f45-fe7d516d56f6.jpg)

![3](https://user-images.githubusercontent.com/39125354/39815317-24391c76-5387-11e8-8a71-6e6645382b14.jpg)
![4](https://user-images.githubusercontent.com/39125354/39815392-5fe20058-5387-11e8-97c1-6c8b861eef70.jpg)

**Comment by SeaHOH** [comment, marked as answer]
换一个文本编辑器，然后重新下载一个 Config.ini。

**Comment by boseliy** [comment]
问题解决了，用了ultraedit编辑了，谢谢！
